frontend/src/pages/analytics/metadata/user-attributes/MetadataUserAttributeSplitPanel.tsx

```diff
@@ -182,6 +189,18 @@ const MetadataUserAttributeSplitPanel: React.FC<
                             loading={loadingUpdateDisplayName}
                             variant="primary"
                             onClick={() => {
+                              if (
+                                attributeDetails.metadataSource ===
+                                MetadataSource.PRESET
+                              ) {
+                                alertMsg(
+                                  t(
+                                    'analytics:valid.metadataNotAllowEditError'
+                                  ),
+                                  COMMON_ALERT_TYPE.Error as AlertType
```

Comment:
1. set alertMsg default alertType in `utils.ts`
```
export const alertMsg = (alertTxt: string, alertType="error") => {
```
2. remove COMMON_ALERT_TYPE, replace it with 'error' | 'warning' | 'info' | 'success';

3. optimize this in all source code.